### PR TITLE
Add Komga 

### PR DESCRIPTION
**Pull request by navanchauhan**

### CODE REVIEW

File: apps.json [hunk 1]
```diff
@@ -1556,6 +1556,16 @@
...
         "url": "https://github.com/YunoHost-Apps/kodi_ynh"
     },
+    "komga": {
+        "branch": "master",
+        "category": "reading",
+        "revision": "HEAD",
+        "state": "working",
+        "subtags": [
+            "comics"
+        ],
```
Comment (alexAubin) [inline apps.json:1566]: That subtag doesn't exists though, so proposing to drop it (or maybe "books" but idk)

**Review by alexAubin — APPROVED** [review]
LGTM apart from small comment about subtag

File: apps.json (hunk 1 above)
Comment (navanchauhan) [reply apps.json:1566]: hmm, sure books is alright for now

Comment (navanchauhan) [reply apps.json]: changed it

**Review by ericgaspar — APPROVED** [review]

**Pull request merged**